src/cms/src/api/emergency-alert/content-types/emergency-alert/lifecycles.js

```diff
@@ -1,23 +1,29 @@
 "use strict";
 
+const format = require('date-fns/format')
+
+const formatDateToPacificTime = (dateString) => {
+  // Strapi returns the date in ISO format e.g. 2025-01-01T00:00:00.000Z
+  // Format the date in YYYY-MM-DD e.g. 2025-01-01
+  return format(dateString, 'yyyy-MM-dd', { timeZone: 'America/Los_Angeles' })
```

Comment:
I don't think `timeZone` is a valid property for the options object ( https://date-fns.org/v2.28.0/docs/format#arguments ?) but it will be using the local timezone anyway. If we need to specify the timezone, there's a plugin for date-fns we could use (or the newer version 4 apparently has timezone functions built in)